projects/doc/HOWTO-contribute.rst

```diff
@@ -20,7 +20,7 @@ It's mostly a set of following commands:
    # usual workflow:
    #   update:
    $ git pull --rebase upstream master
-   #   hax-in teh patch:
```

Comment:
I believe this was intentional by the author of this but I much prefer your change. 🙂